src/__tests__/actions.test.js

```diff
@@ -0,0 +1,15 @@
+import { updateBook } from '../redux/actions';
+import types from '../redux/constants'
+
+describe('actions', function() {
+	it('should create an action with UPDATE_BOOK type and book as payload', () => {
```

Comment:
I personally use prettier always, it helps with problems like this and many others

Reply:
so tabs are mixed with spaces here

Reply:
Prettier is merged. Thx!